<launch>
	<node name="remote_control_handler" pkg="remote_control" type="remote_bridge.py" output="screen">
		<param name="topic_out0" value="axisX" />
		<param name="topic_out1" value="axisY" />
		<param name="channel_out_max0" value="100" />
		<param name="channel_out_min0" value="-100" />
        <param name="channel_out_max1" value="100" />
		<param name="channel_out_min1" value="-100" />
	</node>
	<node name="arcade_drive" pkg="remote_control" type="singleAxisArcadeDrive.py">
		<param name="axisX" value="axisX" />
		<param name="axisY" value="axisY" />
		<param name="axisXMin" value="-100" />
        <param name="axisXMax" value="100" />
        <param name="axisXCtr" value="0" />
        <param name="axisYMin" value="-100" />
        <param name="axisYMax" value="100" />
        <param name="axisYCtr" value="0" />
        <param name="maxPower" value="100" />
        <param name="minPower" value="-100" />
        <param name="ctrPower" value="0" />
		<param name="LeftMotorChannel" value="left" />
        <param name="RightMotorChannel" value="left" />
	</node>
</launch>
